src/libraries/System.Net.Http/tests/UnitTests/Headers/HttpRequestHeadersTest.cs

```diff
@@ -720,8 +720,18 @@ public void From_ReadAndWriteProperty_ValueMatchesPriorSetValue()
             Assert.False(headers.Contains("From"),
                 "Header store should not contain a header 'From' after setting it to null.");
 
-            Assert.Throws<FormatException>(() => { headers.From = " "; });
-            Assert.Throws<FormatException>(() => { headers.From = "invalid email address"; });
+            // values are not validated, so invalid values are accepted
+            headers.From = " ";
```

Comment:
Similarly, since the From header isn't parsed at all anymore, I don't think we need tests specifically for it. We can remove this and the other test below.

Reply:
Actually, never mind. This is testing the strongly typed property, we should keep it.